cellpack/bin/analyze.py

```diff
@@ -33,36 +30,27 @@
 
 class Args(argparse.Namespace):
```

Comment:
To be consistent with our other code, it would be nice to convert this to using Fire, the way `bin/pack` does. Maybe a separate pr?

Reply:
Agreed! I can work on converting this to use the `Fire` package

Reply:
Created #105 to work on this